src/pages/academy/components/_social-sharing.js

```diff
@@ -1,88 +1,144 @@
-import React from 'react'
+import React, { useState } from 'react'
 import styled from 'styled-components'
-import { Flex } from 'components/containers'
+import { url } from './utility'
+import { Show, Container } from 'components/containers'
```

Comment:
```suggestion
import { Desktop, Mobile, Container } from 'components/containers'
```
Please use `Desktop` & `Mobile` instead of `Show`.